Tier Pricing — Added missing docs (#723)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/index.mdx

| Grouped products                | <Badge text="Roadmap" variant="caution" />      |
| Product details                 | <Badge text="Supported" variant="tip" /> |
| Simple product type             | <Badge text="Supported" variant="tip" /> |

> | Tier pricing                    | <Badge text="Supported" variant="tip" /> |

Reviewer: I think we can change this to Product prices (list supported types) similar to the Cart overview:  Product line discounts (catalog rule, special price, tier price) -- not Product line discounts.  (https://developer-stage.adobe.com/commerce/services/reference/rest/#operation/deletePriceBooks)  - regular, discounts, and tiered pricing -- I'm not sure about if special prices is something different than discounts.

In this same table, need to check on the product support -- Catalog service product types were simplified to simple products and complex products --   so maybe just have simple products, complex products-- (See https://developer.adobe.com/commerce/webapi/graphql-api/saas/index.html#definition-ProductView)

Author: Special pricing is different than discounts: https://experienceleague.adobe.com/en/docs/commerce-admin/catalog/products/pricing/pricing-advanced.

Table changes are good. I think I captured them all.

| Virtual product type            | <Badge text="Roadmap" variant="caution" />      |
| Zoom                            | <Badge text="Supported" variant="tip" /> |

## Tier pricing

> Tier pricing allows you to offer quantity-based discounts on your products. When configured in Adobe Commerce, the PDP drop-in automatically displays tier pricing information below the product price, showing customers the savings available when purchasing larger quantities.

Reviewer: Consider this revision 
```suggestion
Tier pricing enables quantity-based discounts that reward customers for buying in bulk. Once configured in Adobe Commerce, the PDP drop-in automatically displays tier pricing below the product price, clearly highlighting the savings available at higher quantities.
```

- **Complex products** (`ComplexProductView`) — Products comprised of multiple simple products with potentially different prices. This includes configurable, bundle, and grouped product types. Complex products return pricing as a [`ProductViewPriceRange`](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/index.html#definition-ProductViewPriceRange) representing the minimum and maximum prices.

For complete API details, see the [`products` query](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/index.html#query-products) in the Merchandising GraphQL API reference.

<Aside type="tip" title="Pro tip!">
  Go to the [Commerce APIs Playground](https://experienceleague.adobe.com/developer/commerce/storefront/playgrounds/commerce-services/) and run the `products` query to see the product data structure for different product types.
</Aside>

<Aside type="note" title="API differences by platform">
  The catalog API varies depending on your Adobe Commerce platform:

>   - **Adobe Commerce Optimizer**: Uses the [Merchandising GraphQL API](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/) with the `products` query endpoint.

Reviewer: ```suggestion
  - **Adobe Commerce Optimizer**: Uses the [Merchandising Services GraphQL API](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/) with the `products` query endpoint.
```

>   - **Adobe Commerce Optimizer**: Uses the [Merchandising GraphQL API](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/) with the `products` query endpoint.

Reviewer: ```suggestion
  - **Adobe Commerce Optimizer**: Uses the [Merchandising GraphQL API](https://developer.adobe.com/commerce/services/optimizer/merchandising-services/using-the-api/) with the `products` query endpoint.
```

>   - **Adobe Commerce Cloud Service**: Uses the [Catalog Service GraphQL API](https://developer.adobe.com/commerce/webapi/graphql/schema/catalog-service/) along with Live Search and Product Recommendations APIs.

Reviewer: ```suggestion
  - **Adobe Commerce as a Cloud Service**: Uses the [Catalog Service GraphQL API](https://developer.adobe.com/commerce/webapi/graphql/schema/catalog-service/) along with Live Search and Product Recommendations APIs.
```
